cli/flox/src/commands/install.rs

```diff
@@ -110,8 +111,14 @@ impl Install {
             ensure_floxhub_token(&mut flox).await?;
         };
 
+        if let ConcreteEnvironment::Managed(ref environment) = concrete_environment {
+            if environment.has_local_changes(&flox)? {
+                bail!(ManagedEnvironmentError::CheckoutOutOfSync)
+            }
+        };
+
```

Comment:
nonblocking: did you mean to add this one?

Reply:
nope, didnt clean that up after we talked

Reply:
fixed